errors/errors.go

```diff
@@ -56,6 +57,11 @@ type BoulderError struct {
 	Type      ErrorType
 	Detail    string
 	SubErrors []SubBoulderError
+
+	// RetryAfter is the number of milliseconds the client should wait before
+	// retrying the request. Do not return this value directly to clients, use
+	// RetryAfterSeconds instead.
+	RetryAfter int64
```

Comment:
I think this should be a time.Duration. That way the only code that has to worry about whether this is seconds or milliseconds is the serialization / deserialization code (isolated within the grpc package).

That would also suggest getting rid of RetryAfterSeconds. When expressing RetryAfter at the HTTP layer, it would be up to the WFE (which is aware of HTTP semantics) how to turn a time.Duration into something that meets the HTTP spec and how to round or truncate it.

Reply:
This also ended up making the wrap/unwrap logic cleaner. Thanks.